src/script/lua_api/l_object.cpp

```diff
@@ -517,11 +518,17 @@ int ObjectRef::l_set_bone_position(lua_State *L)
 	if (sao == nullptr)
 		return 0;
 
-	std::string bone = readParam<std::string>(L, 2, "");
-	v3f position = readParam<v3f>(L, 3, v3f(0, 0, 0));
-	v3f rotation = readParam<v3f>(L, 4, v3f(0, 0, 0));
-
-	sao->setBonePosition(bone, position, rotation);
+	std::string bone;
```

Comment:
log_deprecated missing here and below

Reply:
Inserted the calls at the top of the functions [gs]et_bone_position functions.